examples/phi2/phi2.py

```diff
@@ -124,55 +132,64 @@ def main(raw_args=None):
         legacy_optimization_setting(template_json)
 
     # add pass flows
-    model_type = str(args.model_type)
-    template_json["pass_flows"] = SUPPORTED_WORKFLOWS[model_type]
-    if args.optimum_optimization:
-        # if args.model_type in ("cpu_int4", "cuda_int4"):
-        #     raise ValueError("Int4 optimization is not supported in phi2 optimum optimization")
+    pass_flows = [[]]
+    if args.finetune_method:
+        pass_flows[0].append(args.finetune_method)
+        template_json["systems"]["local_system"]["config"]["accelerators"][0]["device"] = "gpu"
+        # torch fine tuning does not require execution provider, just set it to CUDAExecutionProvider
```

Comment:
I thought ep is not mandatorily required after Mike's PR got merged. Is it still required?

Reply:
If we don't provide an EP, it would loop over the installed EPs which is not what we want.

Reply:
@jambayk will "loop over the installed eps" sill be executed even if we are not running any pass needed onnxruntime like lora/snpe/openvino?

Reply:
Yes, we haven't made the changes that would not do this for workflows with no ort targeting passes. 

Mike's PR didn't really change any behavior of the workflows. It only updated the configs to collect the  hardware/ep related options together.

Reply:
Yes, but the qlora/snpe/openvino pass is EP agnostic, so even with "loop over the installed eps", the pass only run once.

Reply:
The cache does take care of the rerun but still, the looping behavior and multiple footprint/outputs for workflows with no onnx models is not ideal. That's what will be good to improve.